### PR TITLE
Expand the add_peers test

### PR DESCRIPTION
**Pull request by madadam**

### CODE REVIEW

File: tests/integration_tests.rs [hunk 1]
```diff
@@ -232,14 +234,25 @@ fn random_schedule_no_delays() {
 
 #[test]
 fn add_peers() {
+    use parsec::dev_utils::ObservationEvent::*;
+
+    let mut names = NAMES.iter();
     let mut env = Environment::new(SEED);
-    let schedule = Schedule::new(
```
Comment (pierrechevalier83) [inline tests/integration_tests.rs]: I don't know about replacing: I think it's also nice to have one that's more random to possibly catch weird edge cases in soak testing
